e2e/fixtures/mockEventSource.ts

```
@@ -0,0 +1,66 @@
+    }
+
+    close() {}
+  };
+`;
+
+export async function setupEventSourceStub(page: Page, scenario: SseScenario) {
+  const messages = SSE_SCENARIOS[scenario];
+  await page.addInitScript(
+    ({ script, msgs }) => {
+      window.__sseMessages = msgs;
+      eval(script);
```

Biome (2.5.0)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)